src/App.jsx

```diff
@@ -14,70 +15,77 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+const DEFAULT_SELECTED_GOOD = 'Jam';
+const NO_GOOD_SELECTOR = 'No goods selected';
+const IS_SELECT = 'is selected';
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+export const App = () => {
+  const [selectedGood, setSelectedGood] = useState(DEFAULT_SELECTED_GOOD);
 
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
+  return (
+    <main className="section container">
+      {selectedGood ? (
+        <h1 className="title is-flex is-align-items-center">
+          {selectedGood
+            ? `${selectedGood} ${IS_SELECT}`
+            : NO_GOOD_SELECTOR
+          }
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
+          {!!selectedGood && (
             <button
-              data-cy="AddButton"
+              onClick={() => setSelectedGood('')}
+              data-cy="ClearButton"
               type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
-
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Dumplings
-          </td>
-        </tr>
-
-        <tr data-cy="Good" className="has-background-success-light">
-          <td>
-            <button
-              data-cy="RemoveButton"
-              type="button"
-              className="button is-info"
-            >
-              -
-            </button>
-          </td>
-
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Jam
-          </td>
-        </tr>
+              className="delete ml-3"
+            />
+          )}
+        </h1>
+      ) : (
+        <h1 className="title is-flex is-align-items-center">
+          {NO_GOOD_SELECTOR}
+        </h1>
+      )}
 
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
+      <table className="table">
+        <tbody>
+          {goods.map(goodItem => (
+            <tr
+              key={goodItem}
+              data-cy="Good"
+              className={
+                selectedGood === goodItem
```

Comment:
let's create a separate variable for `selectedGood === goodItem` since you're repeating it 2 times.

Don't forget to use `classnames` lib in such cases.